contributors/design-proposals/pod-pid-namespace.md

```diff
@@ -0,0 +1,62 @@
+# Shared PID Namespace for the Docker Runtime
+
+Pods share many namespaces, but the ability to share a PID namespace was not
+supported by Docker until version 1.12. This document proposes how to roll out
+support for sharing the PID namespace in the docker runtime.
```

Comment:
@verb To my reading there aren't any substantive or unaddressed concerns in that thread. You're correct that rkt and cri-o support a shared namespace, but that's incidental rather than something that CRI has already requested